helmfile/values/fluentd-user.yaml.gotmpl

```diff
@@ -10,7 +10,7 @@ elasticsearch:
     enabled: true
     user: fluentd
     password: ""
-  host: elastic.{{ .Values.global.opsDomain }}
+  hosts: elastic.{{ .Values.global.opsDomain }}
```

Comment:
Are you sure this works? It looks like it should be a list to me https://github.com/kokuwaio/helm-charts/blob/main/charts/fluentd-elasticsearch/values.yaml#L66

Reply:
> Are you sure this works? It looks like it should be a list to me https://github.com/kokuwaio/helm-charts/blob/main/charts/fluentd-elasticsearch/values.yaml#L66

I tried both and both works.  I will change to list form to be inline with the documentation.  However, the  list form works without the port number appended. That is the reason why the port is specified in the environment variable below, otherwise, it will be 9200 by default. There seems to be discrepancy in the documentation.